cluster-scope/overlays/moc/zero/kustomization.yaml

```diff
@@ -34,6 +34,8 @@ resources:
   - ../../../base/groups/cluster-admins
   - ../../../base/imageregistryconfigs/cluster
   - ../../../base/ingresscontrollers/default
+  - ../../../base/machineconfigs/ssh-keys-controller
+  - ../../../base/machineconfigs/ssh-keys-worker
```

Comment:
Can we move this into `moc/common` so both clusters are accessible maybe?

Reply:
Sure.